gather.go

```diff
@@ -352,6 +352,11 @@ func (a *Agent) gatherCandidatesSrflx(ctx context.Context, urls []*URL, networkT
 			go func(url URL, network string) {
 				defer wg.Done()
 
+				// if transport protocol is TCP assume UDP calls will fail/timeout.
+				if url.Proto == ProtoTypeTCP {
```

Comment:
if `url.Proto == ProtoTypeTCP`, doesn't even need to start this go routine?

Reply:
This is true, I will make an update to the PR.